ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1002,16 +882,36 @@ describe(RuntimeConfigurationPanel.name, () => {
       workspaceNamespace: workspaceStubs[0].namespace,
       gcePersistentDisk: null,
     });
+
+    mockSetRuntimeRequest = jest.fn();
   });
 
   afterEach(async () => {
     // Some test runtime pooling were interfering with other tests using fake timers helped stopping that
     act(() => clearCompoundRuntimeOperations());
-    jest.clearAllMocks();
+
     getAborter().abort('Unmounting for testing');
-  });
+    jest.clearAllMocks();
```

Comment:
are these all needed, or are they left over from trial and error?

Reply:
No, I was able to delete most of this. Thanks for mentioning it.